GrammarLinter.md

```diff
@@ -0,0 +1,81 @@
+## English Grammar Linter (Vale)
+
+This repository includes `beta` rules based on the [Vale grammar linter](https://docs.errata.ai/vale). To install Vale on a Mac, run:
+
+```
+brew install vale
+```
+
+You can set up Vale with several different IDEs. For this repository,
+I've tested the integration between Vale and VSCode, and described some tips and tricks
```

Comment:
Following the grammar style guides, should we replace "I" with "We" here? The same for line 64

```suggestion
We've tested the integration between Vale and VSCode, and described some tips and tricks
```